gst/rtp/gstrtph264depay.c

```diff
@@ -1213,6 +1247,27 @@ gst_rtp_h264_depay_handle_event (GstRTPBaseDepayload * depay, GstEvent * event)
     case GST_EVENT_FLUSH_STOP:
       gst_rtp_h264_depay_reset (rtph264depay);
       break;
+    case GST_EVENT_CUSTOM_DOWNSTREAM:
```

Comment:
Instead of handling this event yourself, you should better override
```
/* convert the PacketLost event form a jitterbuffer to a GAP event.
 * subclasses can override this.  */
static gboolean
gst_rtp_base_depayload_packet_lost (GstRTPBaseDepayload * filter,
    GstEvent * event)
```

in your class_init:
`klass->packet_lost = gst_rtp_h264_depay_packet_lost;`

with your packet_lost method, and be sure to call the base class method from yours. You would be getting the same GstEvent structure, but in the provisioned way for the depayloaders.

Reply:
done